packages/stores/src/currency-registrar/unsafe-ibc.ts

```diff
@@ -0,0 +1,150 @@
+import { DenomHelper } from "@keplr-wallet/common";
+import { Hash } from "@keplr-wallet/crypto";
+import { ChainStore } from "@keplr-wallet/stores";
+import {
+  AppCurrency,
+  ChainInfo,
+  Currency,
+  IBCCurrency,
+} from "@keplr-wallet/types";
+
+type OriginChainCurrencyInfo = [
+  string, // chain ID
+  string, // coinMinimalDenom
+  { portId: string; channelId: string }[] // IBC path (configured)
+];
+
+/** Will register IBC currencies to the Osmosis chain in the chain store, without querying for IBC trace denom from IBC module.
+ *  It assumes the given IBC asset config is valid.
+ *  Use for major performance boost if working with many IBC assets (as we are on Osmosis). */
+export class UnsafeIbcCurrencyRegistrar<C extends ChainInfo = ChainInfo> {
```

Comment:
Do you think this PR could be a good opportunity to create some basic unit tests for this class in particular? Given the importance of the IBC registrar, I believe implementing these tests would be highly beneficial.

Reply:
I agree, but given the lack of a mock chain store and that Aris had no issues, I'd like to not block the release this week

Reply:
OK I added some tests, chain store was easier to create than I thought